doc/admin/pool_operations.md

```diff
@@ -265,9 +267,31 @@ $ dmg pool exclude --pool=${puuid} --rank=${rank} --target-idx=${idx1},${idx2},$
 The pool target exclude command accepts 3 parameters:
 
 * The pool UUID of the pool that the targets will be excluded from.
-* The rank of the target(s) te be excluded.
+* The rank of the target(s) to be excluded.
 * The target Indices of the targets to be excluded from that rank (optional).
 
+## Pool Drain
+
+Alternatively when an operator would like to remove one or more pool targets they
+can initiate a pool drain operation. A pool drain operation will initiate rebuild
+without excluding the designated target until after the rebuild is complete.
+This allows the target that will be excluded to continue to perform I/O while the rebuild
+operation is ongoing. This also allows non-replicated data to be rebuilt onto another target
```

Comment:
Is the target in a read-only state? If so, might be worth pointing this out. If not, then I don't understand how the target will ever be fully drained on a busy system.

Reply:
The new target and the target being drained should both receive new writes. The rebuild protocol will need to handle the internals of not sending the data written after the rebuild is started. So only the data that exists before the drain started will need to be rebuilt.

Reply:
Ah, ok. That makes sense. Still might be worth explaining this somehow to preemptively address that question so that the reader doesn't have to have studied the rebuild design doc in order to understand that.

Reply:
I agree, I'll add that along with the tests.